src/server/rest/v1/validator/AuthValidator.ts

```diff
@@ -8,7 +8,6 @@ import global from '../../../../types/GlobalType';
 export default class AuthValidator extends SchemaValidator {
   private static instance: AuthValidator|null = null;
   private authSignIn: Schema = JSON.parse(fs.readFileSync(`${global.appRoot}/assets/server/rest/v1/schemas/auth/auth-signin.json`, 'utf8'));
-  private authSignOn: Schema = JSON.parse(fs.readFileSync(`${global.appRoot}/assets/server/rest/v1/schemas/auth/auth-signon.json`, 'utf8'));
```

Comment:
Put everything back in AuthValidator class.

Reply:
Done